requirements/constraints.txt

```diff
@@ -8,6 +8,4 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# edx-toggles is set to 1.2.2 because we need `edx_toggles.toggles.__future__`
-# we also expect v2.0.0 to introduce breaking changes in the feature toggle API
-edx-toggles==1.2.2
+edx-toggles>=2.0.0
```

Comment:
Minimum constraints can go directly in the file with the dependency, and should be removed from this file. Thanks.